.github/workflows/ovc.yml

```diff
@@ -0,0 +1,53 @@
+name: OVC
+on:
```

Comment:
Please, add merge_group trigger also, if you want to make this stage required

Reply:
Done. Please check.